RESULTS.md

```diff
@@ -0,0 +1,61 @@
+Product Test result Summary
+---
+
+Product test results from be2fc0baeda69cad7480e29aa6714b2e5d916c8e tested on 2021-09-30
+
+| Product                           | Passing Jobs  | Failing Jobs  | Skipped Jobs  |
+|-----------------------------------|---------------|---------------|---------------|
+| rocky-boot-iso-x86_64-*           | 0             | 2             | 0             |
+| rocky-minimal-iso-x86_64-*        | 2             | 0             | 0             |
+| rocky-dvd-iso-x86_64-*            | 1             | 6             | 16            |
+| rocky-universal-x86_64-*          | 8             | 24            | 0             |

```

Comment:
Note -  This is currently not real data, I need to update it when my machine finishes running tests.

Reply:
Updated this to reflect real test data